core/toolbox/category.ts

```diff
@@ -225,6 +225,8 @@ export class ToolboxCategory
    */
   protected createContainer_(): HTMLDivElement {
     const container = document.createElement('div');
+    // Ensure that the category has a tab index to ensure it receives focus when
+    // clicked (since clicking isn't managed by the toolbox).
     container.tabIndex = -1;
```

Comment:
Maybe consider adding a test for this & the separator cases in core.

Reply:
Decided to add a test in kbe instead as it was much easier (ran into a lot of problems with click/focus in toolbox's test unfortunately). Didn't bother testing separator as it seems less important.